plugins/ruby-foreman-remote-execution/debian/postinst

```diff
@@ -51,6 +51,10 @@ if [ -f /usr/share/foreman/config/database.yml ]; then
   fi
 fi
 
+PLUGIN_ROOT=`$BUNDLE show $PLUGIN`
+mkdir -p $PLUGIN_ROOT/public
+ln -s /var/lib/foreman/public/webpack/ $PLUGIN_ROOT/public/webpack
```

Comment:
In RPM packaging we have 2 macros:

```
%%%{name}_webpack_plugin %%{gem_instdir}/public/webpack/%%{gem_name}
# foreman_webpack_plugin $PLUGIN_ROOT/public/webpack/$PLUGIN

%%%{name}_webpack_foreman %%{foreman_dir}/public/webpack/%%{gem_name}
# foreman_webpack_foreman /var/lib/foreman/public/webpack/$PLUGIN
```

The symlink is also the other way around. A more equivalent version would be:

```
if [ -e $PLUGIN_ROOT/public/webpack/$PLUGIN ] ; then
  mkdir -p /var/lib/foreman/public/webpack
  ln -snf $PLUGIN_ROOT/public/webpack/$PLUGIN /var/lib/foreman/public/webpack/$plugin`
fi
```

I also don't see the asset precompilation here so I'm wondering if the compiled files even exist.

Reply:
@ekohl The asset compilation occurs in `rules`, and the files are copied there in `install`. I don't mind flipping this around but my knowledge of deb packaging is close to none.